### DISCUSSION TITLE
How to disable "Welcome to podcastfy, your general AI ..." at starting

### DISCUSSION THREAD

**Discussion by toilv** [souzatharsis/podcastfy#196] [Q&A]
Hi Team,
I tried to remove this when podcast creating. Tried using promt to gemini but no luck.

**Comment by souzatharsis** [comment]
Hi,

You can customize the conversation:

```python
conversation_config = {
    "podcast_name": "Artfy Podcast",
    "podcast_tagline": "Because Art is Everywhere"
}
audio_file = generate_podcast(text="your text",
                              conversation_config=conversation_config)
```

See [Conversation Config ](https://github.com/souzatharsis/podcastfy/blob/main/usage/conversation_custom.md).

**Reply by toilv** [reply]
Got it. Thank so much. All settings in conversation_config.yaml

**Reply by Cfomodz** [reply]
@toilv Don't forget to mark this question as answered if it is :)
